test/services/datastream_importer_test.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+# A class to test importing from a Datastream file
+class DatastreamImporterTest < ActiveSupport::TestCase
+  setup do
+    DATASTREAM_FILE = file_fixture('ssg-rhel7-ds.xml')
+  end
+
+  test 'datastream import' do
+    importer = DatastreamImporter.new(DATASTREAM_FILE)
+    importer.expects(:save_benchmark)
+    importer.expects(:save_profiles)
+    importer.expects(:save_rule_references)
+    importer.expects(:save_rules)
+    importer.import!
```

Comment:
What is this meant to test? Currently it just checks that methods like `@op_benchmark.profiles` don't throw an exception

Reply:
It's just testing the right `save_` methods are called, that's it. Each `save_` method is tested elsewhere (in the Xccdf module tests)